wrappers/Python/pytest/test_isoline_tracer.py

```
@@ -0,0 +1,564 @@
+        plot = PropertyPlot('HEOS::R441A.mix', 'TS')
+        limits = plot._get_axis_limits()
+        svals = plot.generate_ranges(plot._x_index, limits[0], limits[1], 120)
+        pressure = 1.0109e7                      # above R441A's cricondenbar
+        tracer = Common.IsoLineTracer(plot._state, CoolProp.iP, CoolProp.iSmass, CoolProp.iP)
+        sat = AbstractState("HEOS", "R441A.mix")
+        shared = AbstractState("HEOS", "R441A.mix")
+        checked = 0
+        for smass in svals:
+            try:
+                tracer.update(pressure, smass)
+            except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)